operator/v1/types_machineconfiguration.go

```diff
@@ -32,6 +32,14 @@ type MachineConfigurationSpec struct {
 	StaticPodOperatorSpec `json:",inline"`
 
 	// TODO(jkyros): This is where we put our knobs and dials
+
+	// managedBootImages allows configuration for the management of boot images for machine
+	// resources within the cluster. This configuration allows users to select resources that should
+	// be updated to the latest boot images during cluster upgrades, ensuring that new machines
+	// always boot with the current cluster version's boot image. When omitted, the ManagedBootImages
+	// feature is essentially disabled.
```

Comment:
I would like to reword this to be a little more direct
```suggestion
	// always boot with the current cluster version's boot image.
	// When omitted, no boot images will be updated.
```